src/NexusMods.App.UI/Pages/LibraryPage/ILibraryItemModel.cs

```diff
@@ -14,75 +14,3 @@ namespace NexusMods.App.UI.Pages.LibraryPage;
 
```

Comment:
The using directives in this file are now unused.
Just clean up in interest of reducing warnings, maybe one day we'll come closer to 0 again.